components/StartForm/StartForm.test.tsx

```diff
@@ -0,0 +1,102 @@
+import StartForm from './StartForm';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { useRouter } from 'next/router';
+
+jest.mock('next/router');
+(useRouter as jest.Mock)
+  .mockReturnValueOnce({
+    query: {},
+  })
+  .mockReturnValueOnce({
```

Comment:
I think these three mocks are for each of the tests? It might be better if each `mockReturnValue` call could be inside the actual test case, not mysteriously up here out of the way